package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prodperfect-keen-tracking",
-  "version": "2.0.9",
+  "version": "2.0.10",
```

Comment:
`npm version patch` should be responsible for bumping versions. It'll tag things as a separate commits.

Reply:
I haven't used this functionality before; when I run it, it requires all changes to be committed. But the `dist` files also contain the version when I run `build`, so it feels like I need to update the version before running the build command; does that mess things up? it'd be like `npm version patch` then `npm run build` and then commit everything and push it up. Does that make sense?